Xamarin.Forms.Core/FlexLayout.cs

```diff
@@ -344,6 +344,8 @@ void OnChildPropertyChanged(object sender, PropertyChangedEventArgs e)
 			if (   e.PropertyName == WidthRequestProperty.PropertyName
 				|| e.PropertyName == HeightRequestProperty.PropertyName) {
 				var item = (sender as FlexLayout)?._root ?? GetFlexItem((BindableObject)sender);
+				if (item == null)
```

Comment:
this code is repeated 4 times, but can't be moved a level higher, as it would imply retrieving the item for all possible property changed.